images/build/debian-iptables/buster/Dockerfile

```diff
@@ -44,3 +43,6 @@ RUN update-alternatives \
 	--install /usr/sbin/ip6tables ip6tables /usr/sbin/iptables-wrapper 100 \
 	--slave /usr/sbin/ip6tables-restore ip6tables-restore /usr/sbin/iptables-wrapper \
 	--slave /usr/sbin/ip6tables-save ip6tables-save /usr/sbin/iptables-wrapper
+
+FROM scratch
+COPY --from=build / /
```

Comment:
neat! do we know that this reduces the size?

Reply:
yes! I've tested :)

gcr.io/k8s-staging-build-image/debian-iptables-amd64     buster-v1.6.3                      780ad1ada900   4 minutes ago        60.2MB

vs

gcr.io/k8s-staging-build-image/debian-iptables-amd64     buster-v1.6.2                      830a8bea5d2d   27 hours ago         87.7MB

Per @thockin suggestion on the original issue :D

Reply:
If we were still using debian-base for other component images, I might point out that some of that size is probably shared layers that can't be shared anymore, but I think here this is probably a reasonable trade-off (with other runtime components being on debian-iptables or distroless, or something else entirely)